airflow/hooks/postgres_hook.py

```diff
@@ -61,13 +62,23 @@ def get_conn(self):
 
     def copy_expert(self, sql, filename, open=open):
         """
-        Executes SQL using psycopg2 copy_expert method
-        Necessary to execute COPY command without access to a superuser
+        Executes SQL using psycopg2 copy_expert method.
+        Necessary to execute COPY command without access to a superuser.
+
+        NOTE: if this method is called with a "COPY FROM" statement
+        and the input file does not exist, it creates an empty file
+        and successfully returns, but no data is loaded actually.
+        So if users want to be aware that the input file does not
+        exist, they have to check its existence by themselves.
         """
-        with open(filename, 'w+') as f:
+        if not os.path.isfile(filename):
+            open(filename, 'w').close()
```

Comment:
nit-picking here: if there is a directory with the same name, this could throw a `IsADirectoryError`.  To make sure file is closed properly, maybe have:

```
with open(filename, 'w'):
  pass
```